rtos/Semaphore.h

```diff
@@ -133,6 +133,19 @@ class Semaphore : private mbed::NonCopyable<Semaphore> {
     */
     osStatus release(void);
 
+    /** Get the number of Semaphore tokens available.
+
+      @note You cannot call this function from ISR context.
```

Comment:
CMSIS-RTOS docs say it can be used from ISR context. We may want to allow that. It's probably actually safer to use from ISR context than application context. Not sure what your use case is.

Reply:
thanks @kjbracey-arm . Now fixed.